sys/riscv/riscv/trap.c

```diff
@@ -76,6 +76,12 @@ __FBSDID("$FreeBSD$");
 #include <sys/dtrace_bsd.h>
 #endif
 
+#if __has_feature(capabilities)
+int log_cheri_exceptions = 0;
+SYSCTL_INT(_machdep, OID_AUTO, log_cheri_exceptions, CTLFLAG_RWTUN,
+    &log_cheri_exceptions, 1, "Print trap frame on CHERI exceptions");
```

Comment:
Should clarify this is for user exceptions only (in both the name and the description)

Reply:
I had just copied this over from MIPS.  Keeping the name the same between the two is probably helpful.  Not sure if this is name is already used in any scripts anywhere?

Reply:
If we change the name we should also update MIPS since then the cheribsd test scripts don't need to use different names per architecture.

Reply:
It's currently used to turn off logging when running cheritest and I think we may also be issuing it in some other scripts

Reply:
Ah, mips uses this toggle for both.  For RISC-V we currently dump registers unconditionally for a CHERI exception since the RISC-V trap handler does that for other kernel exceptions as well.

Reply:
(Also, RISC-V is a bit different here due to merged register file since for RISC-V the generic register dump dumps capabilities always)

Reply:
We should align MIPS to what RISC-V is doing and also change the name at the same time.

Reply:
Ok, I've changed the name and also updated MIPS to match.